hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/impl/MutableStateBase.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.state.impl;
+
+import com.hedera.node.app.spi.state.WritableState;
+import com.swirlds.common.merkle.MerkleNode;
+import java.util.*;
```

Comment:
Do we want to have `*` imports?

Reply:
I do not know. I've always outlawed them in the past. This was done for me automatically by IntelliJ. If we don't want star, we should have a style rule in spotless I think to forbid it.

Reply:
I think there is a setting in IntelliJ as well , to avoid * imports.